src/components/Summary/AlterationSummary.vue

```diff
@@ -7,15 +7,15 @@
     />
 
     <!-- Section Header -->
-    <div class="summary-header">
-      <v-layout row class="mx-0">
-        <v-flex xs9>
+    <div class="summary-header px-4 mb-4">
```

Comment:
Extra bottom margin so the next component below the header has some whitespace. (We don't know which component is next, which is why I couldn't add top margin to the next component.)